src/main/java/seedu/address/model/bid/BidComparator.java

```diff
@@ -0,0 +1,28 @@
+package seedu.address.model.bid;
+
+import java.util.Comparator;
+
+public class BidComparator implements Comparator<Bid> {
+
+    @Override
+    public int compare(Bid bidOne, Bid bidTwo) {
+
+        int bidOnePropertyId = bidOne.getPropertyId().getId();
```

Comment:
I think instead of this, maybe can make Id and Price comparable. So just need to sort by id first then sort by price. Perhaps can keep this first, I don't mind making Id and Price Comparable.